packages/api/src/image-info.ts

```diff
@@ -27,6 +27,11 @@ export interface ImageInfo extends Dockerode.ImageInfo {
   History?: string[];
   Digest: string;
   isManifest?: boolean;
+  // Do not use isManifestList. It was added in >5.0.2 of podman and is here for compatibility reasons,
+  // use isManifest instead in the API.
+  // We add isManifestList to be able to retrieve it from the API podmanListImages function call as it returns <unknown>
+  // note that isManifestList is purposely not added to the extension-api.d.ts file as this is only for internal use.
+  isManifestList?: boolean;
```

Comment:
why are we not removing this property instead of saying 'do not use' ?

Reply:
Sorry, I'm confused.

We have to add isManifestList so we can get use the value when calling libpod.listImages, as it's <unknown> return and we rely on ImageInfo to retrieve all information.

We also have to keep `isManifest` since extensions (bootc) and other parts of Podman Desktop use it. 

That's why isManifestList is in image-info.ts, but not in the extensions api.

Reply:
NVM! You are right.

We do not need it as we use:

[let fetchedImages: any[] = [];](https://github.com/cdrage/podman-desktop/blob/6c9c331bcf11e8e91df3c150ebc18944ac2e89dd/packages/main/src/plugin/container-registry.ts#L626-L627)

In the podmanListImages call!

I'll make the change now.